enterprise/cmd/frontend/internal/batches/resolvers/resolver_test.go

```diff
@@ -1109,6 +1111,21 @@ func TestListChangesetOptsFromArgs(t *testing.T) {
 				RepoID: repoID,
 			},
 		},
+		// CheckOpenOrDraft changesets
+		{
+			args: &graphqlbackend.ListChangesetsArgs{
+				CheckOpenOrDraft: &checkOpenOrDraft,
+			},
+			wantSafe: true,
+			wantParsed: store.ListChangesetsOpts{
+				PublicationState: &checkOpenOrDraftPublicationState,
```

Comment:
This already exists for the other tests above :) 

```suggestion
				PublicationState: &wantPublicationStates[0],
```

Reply:
Oh! Thanks man.